prow/tide/status.go

```diff
@@ -220,8 +223,21 @@ func requirementDiff(pr *PullRequest, q *config.TideQuery, cc contextChecker) (s
 // in order to generate a diff for the status description. We choose the query
 // for the repo that the PR is closest to meeting (as determined by the number
 // of unmet/violated requirements).
-func expectedStatus(queryMap *config.QueryMap, pr *PullRequest, pool map[string]PullRequest, cc contextChecker) (string, string) {
+func expectedStatus(queryMap *config.QueryMap, pr *PullRequest, pool map[string]PullRequest, cc contextChecker, blocks blockers.Blockers) (string, string) {
 	if _, ok := pool[prKey(pr)]; !ok {
+		// if the branch is blocked forget checking for a diff
+		blockingIssues := blocks.GetApplicable(string(pr.Repository.Owner.Login), string(pr.Repository.Name), string(pr.BaseRef.Name))
+		var numbers []string
+		for _, issue := range blockingIssues {
+			numbers = append(numbers, strconv.Itoa(issue.Number))
+		}
+		if len(numbers) > 0 {
+			var s string
+			if len(numbers) > 1 {
+				s = "s"
+			}
+			return github.StatusPending, fmt.Sprintf(statusNotInPool, fmt.Sprintf(" Merging is blocked by issue%s %s.", s, strings.Join(numbers, ", ")))
```

Comment:
Lets use `github.StatusError` or `github.StatusFailure`.  This should be red 😈 😈 😈